3.extract_features/pipelines/OP_pipe.cppipe

```diff
@@ -0,0 +1,199 @@
+    Calculate custom percentiles:No
+    Specify percentiles to measure:10,90
+
+MeasureObjectNeighbors:[module_num:9|svn_version:'Unknown'|variable_revision_number:3|show_window:False|notes:['This measures adjacent nuclei neighbors.']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
+    Select objects to measure:ConvertImageToObjects
+    Select neighboring objects to measure:ConvertImageToObjects
+    Method to determine neighbors:Adjacent
+    Neighbor distance:5
+    Consider objects discarded for touching image border?:Yes
+    Retain the image of objects colored by numbers of neighbors?:No
+    Name the output image:ObjectNeighborCount
+    Select colormap:Blues
+    Retain the image of objects colored by percent of touching pixels?:No
+    Name the output image:PercentTouching
+    Select colormap:Oranges
+
+MeasureObjectIntensityDistribution:[module_num:10|svn_version:'Unknown'|variable_revision_number:6|show_window:False|notes:['This module measures the intensity distribution within the objects using the center x,y coordinates for each object (not using the other options as it seems better to measure within the whole objects unlike what happens in the other options).']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
+    Select images to measure:OP
+    Hidden:1
+    Hidden:1
+    Hidden:0
+    Calculate intensity Zernikes?:Magnitudes and phase
+    Maximum zernike moment:9
+    Select objects to measure:ConvertImageToObjects
+    Object to use as center?:These objects
+    Select objects to use as centers:None
+    Scale the bins?:Yes
+    Number of bins:4
+    Maximum radius:100
+
+MeasureObjectSizeShape:[module_num:11|svn_version:'Unknown'|variable_revision_number:3|show_window:False|notes:['This measures the size and shape (among other measurements) for the objects.']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
+    Select object sets to measure:ConvertImageToObjects
+    Calculate the Zernike features?:Yes
+    Calculate the advanced features?:Yes
+
+MeasureTexture:[module_num:12|svn_version:'Unknown'|variable_revision_number:7|show_window:False|notes:['This module will measure the texture within the objects and whole images.']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
+    Select images to measure:OP
+    Select objects to measure:ConvertImageToObjects
+    Enter how many gray levels to measure the texture at:256
+    Hidden:1
+    Measure whole images or objects?:Both
+    Texture scale to measure:3
+
+ExportToDatabase:[module_num:13|svn_version:'Unknown'|variable_revision_number:28|show_window:False|notes:['This module outputs an sqlite file with all of the image and object features for eahc plate in individual folders.', '', 'The previous module to measure image quality was removed since image quality should be run on raw, uncorrected images.']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
```

Comment:
```suggestion
ExportToDatabase:[module_num:13|svn_version:'Unknown'|variable_revision_number:28|show_window:False|notes:['This module outputs an sqlite file with all of the image and bone features.',]|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
```
Recommend updating these comments to be more related to this project in the GUI.